src/index.html

```diff
@@ -4,8 +4,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Webpack Exercise</title>
+    <script src="https://kit.fontawesome.com/2b1476549e.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <header></header>
+    <main>
+        <ul class="list-body">
+            <li class="title">
+                <h2>Today's To Do</h2>
+                <i class="fas fa-sync-alt"></i>
+            </li>
+            <li class="type-list">
+                <input class="text" type="text" placeholder="Add to your list...">
+                <i class="fa fa-share" aria-hidden="true"></i>
+            </li>
+            <li class="list-section"></li>
+            <li class="clear-completed">
+                <button type="button" id="clear">Clear all completed</button>
+            </li>
+        </ul>
+    </main>
+    <footer></footer>
```

Comment:
- [ ] Unfortunately, your data is not being rendered here. Please try to fix this
![Screenshot from 2022-02-16 14-55-21](https://user-images.githubusercontent.com/62705947/154279443-99cdfaa7-a25e-49d7-a6aa-493337807600.png)